lib/game.rb

```diff
@@ -100,7 +97,7 @@ def console
   end
 
   def shoot
-    if xy = convert
+    if xy == convert
```

Comment:
No man. This was intended to be assignment and comparison in one step. Did you actually play in this game after you made your changes?